src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx

```diff
@@ -198,19 +210,28 @@ class Issues extends React.Component<Props, State> {
   }
 }
 
+// used in media query
 const FilterButton = styled(DropdownButton)``;
 const DiscoverButton = styled(Button)``;
+const IssuesButton = styled(Button)``;
 
 const ControlsWrapper = styled('div')`
   display: flex;
   align-items: center;
   justify-content: space-between;
   margin-bottom: ${space(1)};
-  @media (max-width: ${p => p.theme.breakpoints[0]}) {
-    ${FilterButton}, ${DiscoverButton} {
+  @media (max-width: ${p => p.theme.breakpoints[2]}) {
+    ${FilterButton}, ${DiscoverButton}, ${IssuesButton} { /* stylelint-disable-line selector-type-no-unknown */
```

Comment:
How do I fix the 
`224:35  ✖  Unexpected unknown type selector "$dummyValue"   selector-type-no-unknown` error without disabling the stylelint?

I believe that we are helping the linter somewhere in the codebase, but can't remember where.